### PR TITLE
Unit tests for oauth.callback.OAuthCallbackManager

### PR DESCRIPTION
**Pull request by madumalt**
### Proposed changes in this pull request
- Unit tests for Unit tests for oauth.callback.OAuthCallbackManager

### When should this PR be merged
- ASAP

### Follow up actions
- Trigger a Jenkins build

### CODE REVIEW

File: components/org.wso2.carbon.identity.oauth/src/test/java/org/wso2/carbon/identity/oauth/callback/OAuthCallbackManagerTest.java [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+        OAuthCallbackManager oAuthCallbackManager = new OAuthCallbackManager();
+        AuthenticatedUser authenticatedUser = new AuthenticatedUser();
+        OAuthCallback oAuthCallback = new OAuthCallback(authenticatedUser, "client", OAuthCallback.OAuthCallbackType
+                .ACCESS_DELEGATION_AUTHZ);
+        oAuthCallback.setAuthorized(false);
+
+        oAuthCallbackManager.handleCallback(oAuthCallback);
+    }
+}
```
Comment (nuwandiw) [inline components/org.wso2.carbon.identity.oauth/src/test/java/org/wso2/carbon/identity/oauth/callback/OAuthCallbackManagerTest.java]: Add new line at the end file

File: components/org.wso2.carbon.identity.oauth/src/test/java/org/wso2/carbon/identity/oauth/callback/OAuthCallbackManagerTest.java [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+ */
+@PrepareForTest({OAuthCallbackHandlerRegistry.class})
+public class OAuthCallbackManagerTest extends PowerMockTestCase {
+
+    @Mock
+    private OAuthCallbackHandlerRegistry oAuthCallbackHandlerRegistry;
+
+    @Test
+    public void testHandleCallbackNoCallBackHandlerCanbeFound() throws Exception {
+
```
Comment (nuwandiw) [inline components/org.wso2.carbon.identity.oauth/src/test/java/org/wso2/carbon/identity/oauth/callback/OAuthCallbackManagerTest.java]: Remove new line after method signature

Comment (madumalt) [reply components/org.wso2.carbon.identity.oauth/src/test/java/org/wso2/carbon/identity/oauth/callback/OAuthCallbackManagerTest.java]: We all have agreed to add a new line after the method signature. 
So this is the conventions.

File: components/org.wso2.carbon.identity.oauth/src/test/java/org/wso2/carbon/identity/oauth/callback/OAuthCallbackManagerTest.java (hunk 1 above)
Comment (madumalt) [reply components/org.wso2.carbon.identity.oauth/src/test/java/org/wso2/carbon/identity/oauth/callback/OAuthCallbackManagerTest.java]: done

**Pull request merged**